src/containers/PipelineResources/PipelineResources.js

```diff
@@ -13,19 +13,16 @@ limitations under the License.
 
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { Link } from 'react-router-dom';
+import { injectIntl } from 'react-intl';
 import {
   InlineNotification,
-  StructuredListBody,
-  StructuredListCell,
-  StructuredListHead,
-  StructuredListRow,
-  StructuredListSkeleton,
-  StructuredListWrapper
+  StructuredListSkeleton
 } from 'carbon-components-react';
-import { getErrorMessage, urls } from '@tektoncd/dashboard-utils';
+import { getErrorMessage } from '@tektoncd/dashboard-utils';
 
+import { PipelineResources as PipelineResourcesList } from '@tektoncd/dashboard-components';
 import { fetchPipelineResources } from '../../actions/pipelineResources';
+import { deletePipelineResource } from '../../api';
```

Comment:
Sidenote, I expected this to be an action but see this is how the current deletePipelineRun code works too and we could do that move separately (and display notifications accordingly, emulating the existing secrets behaviour)